proposals/sar_backscatter.json

```
@@ -0,0 +1,78 @@
+{
+    "id": "sar_backscatter",
+    "summary": "Computes backscatter from georeferenced SAR input",
+    "description": "Computes backscatter from georeferenced SAR input.\n\nNote that backscatter computation may require instrument specific metadata that is tightly coupled to the original SAR products. As a result, this process may only work in combination with loading data from specific collections, not with general data cubes.",
+    "categories": [
+        "cubes",
+        "sar"
+    ],
+    "experimental": true,
+    "parameters": [
+        {
+            "description": "Data cube containing raw backscatter input.",
+            "name": "data",
+            "schema": {
+                "subtype": "raster-cube",
+                "type": "object"
+            }
+        },
+        {
+            "description": "The backscatter coefficient to compute, either `sigma0` or `gamma0` (the default).",
+            "name": "backscatter_coefficient",
+            "optional": true,
+            "default": "gamma0",
+            "schema": {
+                "type": "string",
+                "enum": [
+                    "sigma0",
+                    "gamma0"
+                ]
+            }
+        },
+        {
+            "description": "By default, orthorectification is disabled. Set to `true` to enables orthorectification.\n\nThe non-orthorectified products use a simple earth model as provided in the products themselves. This may be sufficient for very flat target areas and is faster to process.",
```

> The non-orthorectified products use a simple earth model as provided in the products themselves.

This also sounds like a (very basic) orthorectification, but it says also that false disables orthorectification. That is a bit confusing.

Also, I added a typo here: ;-)
```suggestion
            "description": "By default, orthorectification is disabled. Set to `true` to enable orthorectification.\n\nThe non-orthorectified products use a simple earth model as provided in the products themselves. This may be sufficient for very flat target areas and is faster to process.",
```

---

@lforesta could you help here?

---

@m-mohr I'm not sure what's the question, if this should be `true` or `false` by default? The S1 ARD data we generate with SNAP/pyroSAR have orthorectification (actually two different topographic corrections are used, maybe @clausmichele can add details), so I would set `true` as default.

---

@lforesta Thank you. The question is whether this part of the process description "The non-orthorectified products use a simple earth model as provided in the products themselves." is also some kind of orthorectification? Then it would be a bit strange to say "orthorectification" is disabled.

---

Ah I see. If it's disabled, no DEM is used to take into account the topography, and hence I would not call the data orthorectified. Imho it would be geocoding/georeferencing without orthorectification.
What do @clausmichele @aljacob @przell think about this?

---

I guess orthorectification is called terrain-correction in SNAP. So disabling should be possible also in SNAP. Then it will not be CARD4L compliant though. So in the genereic sar_backscatter process we can keep it as a parameter and in the ard_generation process we will set it to "true".

---

Thank you. Solved.